lib/screens/v2/candidate_generator/dup.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Screens.V2.CandidateGenerator.Dup do
+  @moduledoc false
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.Dup
+  alias Screens.Config.V2.Header.CurrentStopId
+  alias Screens.Stops.Stop
+  alias Screens.V2.CandidateGenerator
+  alias Screens.V2.Template.Builder
+  alias Screens.V2.WidgetInstance.{NormalHeader, Placeholder}
+
+  @behaviour CandidateGenerator
+
+  @impl CandidateGenerator
+  def screen_template do
+    {:screen,
+     %{
+       screen_normal: [
+         {:rotation_zero,
+          %{
+            body_normal: [
+              :header,
+              :main_content_primary,
+              :inline_alert
+            ],
+            body_takeover: [:full_body]
+          }},
+         {:rotation_one,
+          %{
+            body_normal: [
+              :header,
+              :main_content_primary,
+              :inline_alert
+            ],
+            body_takeover: [:full_body]
+          }},
+         {:rotation_two,
+          %{
+            body_normal: [
+              :header,
+              :main_content_secondary,
```

Comment:
I'm a bit fuzzy on the way the dup template works. I'd guess that 0 and 2 should be `main_content_primary` if they are supposed to hold the same main content, and 1 would be `main_content_secondary`, but I might be missing the mark here?

Reply:
V1 DUPs currently have 0 and 1 showing the same list of departures, not 0 and 2. This is at least true for departures. I do think I can see an issue with this current layout when thinking about alerts. While 0 and 1 share logic for departures, they do not share logic for alerts that are not `:inside`. For `:boundary` alerts, 0 shows a partial and 1 shows a fullscreen. 

I _think_ we might be able to handle this by giving each rotation their own `full_body_rotation_x` in the `body_takeover` part of the layout. Then in the `CandidateGenerator` logic for `Alert`s, we can flag fullscreen vs inline on the candidate.

Reply:
We may actually be able to get rid of the `:inline_alert` for rotation 2. I see anything in the code that says that rotation shows an inline alert for live alerts, only overrides. And since overrides are being skipped, it may never get used.

Reply:
I decided to try this out. Each rotation now has its own alert slot if it will need one (I couldn't find any instance of v1 DUP showing a partial alert on rotation one, so I removed it from the layout). Each rotation also now has a `full_screen_takeover` slot. This will likely always be used for full screen alerts, but evergreen could also use it.

Reply:
For the reason why we did the primary departures on 0 and 1, it was just because it was a little easier to group them together like that in code. The end result on the screens is the same, since they cycle through the 3 pages repeatedly:

`A- A-B-A- A-B-A- A-B ...`
vs, if primary departures were on 0 and 2:
`   A-B-A- A-B-A- A-B-A ...`